Added credential storage automation documentation

ddybvig-perf opened the pull request:
Here it is. Mostly just made a table for the subcommands.

ddybvig-perf committed:
Added credential storage automation documentation

vossen-adobe reviewed (changes requested):
I think we should put this table in the actual cred man documentation in additional tools.  In this section, we can put some usage example which falls inline with best practice.  We'll also want to update the paragraph above that that discusses keyring options, and add the cryptfile to the list.  We should also say something about which ones are used by default on which platform.

vossen-adobe committed:
additional_tools

ddybvig-perf committed:
Merge remote-tracking branch 'adobe-dmeservices/additional_tools' into credman-docs

ddybvig-perf committed:
removed table from deployment_best_practices to additional_tools

vossen-adobe committed:
Revise description

vossen-adobe reviewed (changes requested):
I think we should add some simple usage scenario for set/get.  Like: user_sync credentials set ldap_password password or similar.   Show the output from the command, and then do the same for get.

Once we add the automation piece, it will be good to tie it all together

ddybvig-perf committed:
small heading additions. need to check out other branch to generate sample output

ddybvig-perf committed:
added sample outputs for get and set
